### DISCUSSION TITLE
截止到目前code=45还是没有解决方法吗

### DISCUSSION THREAD

**Discussion by AlphaNight001** [simple-robot/Discussions#135] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

启动项目后code45，只能换一个别的QQ去使用了

**Comment by ForteScarlet** [comment]
各种解决办法参考 https://github.com/mamoe/mirai/issues/2570 ，但是要根除还得等mirai更新

**Reply by AlphaNight001** [reply]
mirai-login-solver-sakura-0.0.10.mirai2
你知道这个怎么用吗，并不是很了解这种插件的安装

**Reply by ForteScarlet** [reply]
如果说的是 mirai-console 插件的话，那需要使用 `mirai-console` 。simbot依赖的是 `mirai-core` ，不能加载 mirai-console 的插件

**Comment by codez-yydd** [comment]
有解决喊一下我，我三个号全是45，之前运行了一月然后出了问题，我用mirai的扫码登录了另外一个账号，只运行了一天不到，现在无论哪个号都是45，运行程序的电脑和挂qq的手机都是同一个ip，全都是45

**Reply by yuyang0403** [reply]
我现在也是，5个号全军覆没
